### DISCUSSION TITLE
Build works but shows error saying "must have an implementation part because it has accessibility modifiers"

### DISCUSSION THREAD

**Discussion by jackdh** [riok/mapperly#374] [Q&A] [answered]
Hey,

Amazing library, have a bit of werid issue in that it builds okay, however I have a error in the IDE saying:

"must have an implementation part because it has accessibility modifiers"

![image](https://user-images.githubusercontent.com/1907451/234277084-1abddfc6-ee1e-4508-8178-ba2e2113cd4d.png)
Everything works okay, but not sure if I've done something wrong here?

**Comment by TimothyMakkison** [comment]
I don't think you've fully censored the Source object

**Reply by jackdh** [reply]
Ha, all well it doesn't really matter!

**Comment by CommonGuy** [comment, marked as answer]
I think the error shows because at first, the method really does not have an implementation. It will be generated by the source generator.

Seems like a bug in the IDE that it does not pick up the generated implementation after the build.

**Comment by Kog008** [comment]
I faced this behaviour several times. Restart Visual Studio solved it every time.
